src/components/pages/Homepage/Homepage.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
 import { ImageButton } from '../../../components';
-import { useHistory } from 'react-router-dom';
+//import { useHistory } from 'react-router-dom';
```

Comment:
Please revert this commit, it overwrites some changes I made to this file. To do so, you can checkout the file from master and then commit to your branch